clients/drcachesim/tools/opcode_mix.h

```diff
@@ -83,7 +84,7 @@ class opcode_mix_t : public analysis_tool_t {
 
 protected:
     struct worker_data_t {
-        std::unordered_map<app_pc, int> opcode_cache;
+        std::unordered_map<app_pc, std::pair<int, uint>> opcode_category_cache;
```

Comment:
(Optional (wish Github had a way to mark things as optional): use a new struct with meaningful names rather than std::pair.  This is not in our Github style guide so optional.)

Reply:
Done.